app/models/article.rb

```diff
@@ -4,4 +4,8 @@ class Article < ApplicationRecord
   def title?
     title.present?
   end
+
+  def has_title
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/sample-rails-ruby/blob/37b6be83debe6479d73dd46d1921c693915fed7f/app/models/article.rb#L8"></span>Naming/PredicateName: Rename <code>has_title</code> to <code>title?</code>. (https://github.com/rubocop-hq/ruby-style-guide#bool-methods-qmark)</td>
    </tr>
  </tbody>
</table>